src/core/agent/services/signifyNotificationService.test.ts

```diff
@@ -1003,28 +1226,18 @@ describe("Long running operation tracker", () => {
       expect((error as Error).message).toBe("Force Exit");
     }
     expect(operationsGetMock).toBeCalledTimes(1);
-    expect(Agent.agent.ipexCommunications.markAcdcComplete).toBeCalledTimes(0);
+    expect(Agent.agent.ipexCommunications.markAcdc).toBeCalledTimes(0);
     expect(operationPendingStorage.deleteById).toBeCalledTimes(1);
   });
 
   test("Should call setTimeout listening for pending operations if Keria is offline", async () => {
     const callback = jest.fn();
     Agent.agent.getKeriaOnlineStatus = jest.fn().mockReturnValueOnce(false);
-    operationPendingGetAllMock.mockResolvedValueOnce([
```

Comment:
Not sure if there was a merge issue but we should keep this and `expect(operationsGetMock).not.toBeCalled();`